scopes/workspace/workspace/workspace-component/workspace-component-loader.ts

```diff
@@ -233,9 +236,12 @@ export class WorkspaceComponentLoader {
     );
   }
 
-  private async executeLoadSlot(component: Component) {
+  private async executeLoadSlot(component: Component, opts?: { loadDocs?: boolean }) {
     const entries = this.workspace.onComponentLoadSlot.toArray();
     const promises = entries.map(async ([extension, onLoad]) => {
+      if (opts?.loadDocs === false && extension === DocsAspect.id) {
+        return;
+      }
```

Comment:
I would pass the options to 
OnComponentLoad function (see scopes/workspace/workspace/on-component-events.ts)
and then move this condition to the docs.main.runtime
The docs aspect (or any other) might do different things during the on load. 
maybe some of them are still needed when this flag is false.
The only one that knows it, is the aspect the running the onLoad function.

Reply:
done